mobius/src/quantized_hash.rs

```diff
@@ -0,0 +1,9 @@
+use std::hash::Hash;
+
+/// Trait for a type that can be quantized to form a
+/// hash key. The hash key may depend on the quantization
+/// bits.
+pub trait QuantizedHash {
```

Comment:
At this point, is this actually needed? Complex certainly can have a `quantize()` function, but not sure I need this hashable type just yet.